addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -108,6 +111,8 @@ QUnit.module('relational_fields', {
                         ["product", "Product"], ["partner", "Partner"]]},
                     product_id: {string: "Product", type: "many2one", relation: 'product', required: true},
                     partner_ids: {string: "Partner", type: "many2many", relation: 'partner'},
+                    o: {string: "Product", type: "one2many", relation: 'product'},
```

Comment:
product_ids and user_ids

Also, maybe add all those fields only in your test, s.t. it doesn't "pollute" the common models